test/commands/scanner/e2e.cpd.test.ts

```diff
@@ -11,238 +12,133 @@ const Vf_File2 = path.join(Cpd_Test_Code_Path, "myVfPage2.page");
+					verifyEnvVarIsUsedForMinimumTokens(output.shellOutput);
 				});
 
-				after(() => {
-					delete process.env['SFDX_SCANNER_CPD_MINIMUM_TOKENS'];
+				it("...a partly numeric string", () => {
+					// The environment variable processing will strip non-numeric characters, making this "50".
+					process.env[MINIMUM_TOKENS_ENV_VAR] = 'My5String0';
+					const output = runCommand(`scanner run --target ${Cpd_Test_Code_Path} --engine cpd --format json`);
+					verifyEnvVarIsUsedForMinimumTokens(output.shellOutput);
 				});
-
-				setupCommandTest
-					.command([
-						"scanner:run",
-						"--target", Cpd_Test_Code_Path,
-						"--engine", ENGINE.CPD,
-						"--format", "json"
-					])
-					.it("Uses config value when environment variable is not a number", ctx => {
-						verifyDefaultConfigIsUsedForMinimumTokens(ctx);
-					});
 			});
 
-			describe("Invalid empty Minimum tokens value from env", () => {
-				before(() => {
-					process.env['SFDX_SCANNER_CPD_MINIMUM_TOKENS'] = '';
+			describe("Pulls value from config if environment variable is...", () => {
+				it("...undefined", () => {
+					delete process.env[MINIMUM_TOKENS_ENV_VAR];
+					const output = runCommand(`scanner run --target ${Cpd_Test_Code_Path} --engine cpd --format json`);
+					verifyDefaultConfigIsUsedForMinimumTokens(output.shellOutput);
 				});
 
-				after(() => {
-					delete process.env['SFDX_SCANNER_CPD_MINIMUM_TOKENS'];
+				it("...a wholly non-numeric string", () => {
+					process.env[MINIMUM_TOKENS_ENV_VAR] = 'Some String';
+					const output = runCommand(`scanner run --target ${Cpd_Test_Code_Path} --engine cpd --format json`);
+					verifyDefaultConfigIsUsedForMinimumTokens(output.shellOutput);
 				});
 
-				setupCommandTest
-					.command([
-						"scanner:run",
-						"--target", Cpd_Test_Code_Path,
-						"--engine", ENGINE.CPD,
-						"--format", "json"
-					])
-					.it("Uses config value when environment variable is not empty", ctx => {
-						verifyDefaultConfigIsUsedForMinimumTokens(ctx);
-					});
-			});
-
-			describe("String and digit Minimum tokens value from env", () => {
-				before(() => {
-					process.env['SFDX_SCANNER_CPD_MINIMUM_TOKENS'] = 'My5String0';
+				it('...an empty string', () => {
+					process.env[MINIMUM_TOKENS_ENV_VAR] = '';
+					const output = runCommand(`scanner run --target ${Cpd_Test_Code_Path} --engine cpd --format json`);
+					verifyDefaultConfigIsUsedForMinimumTokens(output.shellOutput);
 				});
-
-				after(() => {
-					delete process.env['SFDX_SCANNER_CPD_MINIMUM_TOKENS'];
-				});
-
-				setupCommandTest
-					.command([
-						"scanner:run",
-						"--target", Cpd_Test_Code_Path,
-						"--engine", ENGINE.CPD,
-						"--format", "json"
-					])
-					.it("Uses config value when environment variable is not a number", ctx => {
-						verifyEnvVarIsUsedForMinimumTokens(ctx);
-					});
 			});
-
-		});
-
-		describe("Violation message content", () => {
```

Comment:
Is this test dropped or was it moved somewhere? It looked like this test was running the entire test code path as a target... is that why it wasn't worth it (it was too broad of a test)?

Reply:
This test and [the one that used to be at line 46](https://github.com/forcedotcom/sfdx-scanner/pull/1241/files/22b1284e52a147ee405e897d4ea65ed27c173ab6#diff-59a0f4c9e9d22c88d6ee57aa027bf224f3956e10309ccd5bad1407e5ce411be0L46) were running the same command, so I consolidated them into [the one that's now at line 33](https://github.com/forcedotcom/sfdx-scanner/pull/1241/files/22b1284e52a147ee405e897d4ea65ed27c173ab6#diff-59a0f4c9e9d22c88d6ee57aa027bf224f3956e10309ccd5bad1407e5ce411be0R32).